www/waterfall_view/src/module/main.module.coffee

```diff
@@ -470,6 +470,7 @@ class Waterfall extends Controller
             else "150,0 170,#{height / 2} 150,#{height} 0,#{height} 0,0"
         tooltip = e.append('g')
             .attr('class', 'svg-tooltip')
+            .style('pointer-events', 'none')
```

Comment:
I'd rather put this style in styles.less
```css
.svg-tooltip {
pointer-events: 'none';
}
```

Reply:
OK, I changed it